src/DeserializationPojo/CreateResponsePojo.java

```diff
@@ -0,0 +1,23 @@
+package DeserializationPojo;
```

Comment:
Use better commit names. The general idea is to group a logically similar set of changes in a single commit and name the commit accordingly. For e.g. you could have a commit for adding and using pojo class for create user api and then the commit message would say exactly that. A rather long read, but read it when you have time: https://sethrobertson.github.io/GitBestPractices/